n64.mk

```diff
@@ -52,7 +52,7 @@ CFLAGS+=-MMD     # automatic .d dependency generationc
 CXXFLAGS+=-MMD     # automatic .d dependency generationc
 ASFLAGS+=-MMD    # automatic .d dependency generation
 
-N64_CXXFLAGS := $(N64_CFLAGS) -std=c++11
```

Comment:
There might be existing code depending on this. I think we can change it in a way that the consumer can override without changing the default